docs/dev/run-nb-test.py

```diff
@@ -41,6 +42,8 @@ def main(cli_profile, workspace_tmp_dir, dbfs_init_script_dir, source_dir, nbs):
     work_dir = os.path.join(workspace_tmp_dir, identifier)
     with open(JOBS_JSON, 'r') as f:
         jobs_json = json.load(f)
+    with open(JOBS_HAIL_JSON, 'r') as f:
+        jobs_hail_json = json.load(f)
```

Comment:
With the diff format, this would look like: `hail_jobs_json = base_jobs_json.update(json.load(f))`